src/__tests__/App.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
```

Comment:
I can see all your snapshot tests are passing. 👏👏👏
<img width="339" alt="Screen Shot 2022-10-05 at 09 20 44" src="https://user-images.githubusercontent.com/69011963/193980768-7ac70bbf-0b20-4eab-baf2-dcd01710e3c2.png">

However, it's not as expected unless fixing up some shown errors in your console. You can take a look at these examples:

`App.test.js`

<img width="1189" alt="Screen Shot 2022-10-05 at 09 20 37" src="https://user-images.githubusercontent.com/69011963/193980864-e37ee3fd-9dcf-4e00-9cf7-55a5ca5ebe32.png">